CMakeLists.txt

```diff
@@ -77,6 +78,20 @@ find_package(FFTW3 REQUIRED)
 include_directories(${FFTW3_INCLUDES})
 
 ########################################################################
+#Process GROMACS settings
+########################################################################
+
+if(WITH_XTC)
+find_package(GROMACS CONFIG)
+include_directories(${GROMACS_INCLUDE_DIRS})
+  if(GROMACS_FOUND)
+    add_definitions( -DHAS_GROMACS )
+  else(GROMACS_FOUND)
+    set(WITH_XTC OFF)
```

Comment:
I am not sure if that can be done! `WITH_XTC` is an cached option, you would need to to use `set(...FORCE)` to do that. Plus, it is a bit confusing for the user, which explicitly enabled the option.